judgments/tests.py

```diff
@@ -396,13 +396,14 @@ def test_min_max():
 
 
 def test_prep_query():
-    assert utils.remove_unquoted_stop_words("weight of evidence") == "weight evidence"
-    assert (
-        utils.remove_unquoted_stop_words("'weight of evidence'")
-        == "'weight of evidence'"
-    )
+    # Stopwords outside quoted strings are removed.
+    assert utils.preprocess_query("weight of evidence") == "weight evidence"
+    # Stopwords inside quoted strings are removed
+    assert utils.preprocess_query('"weight of evidence"') == '"weight of evidence"'
+    # Curly quotes are normalised to straight ones and treated as such:
+    assert utils.preprocess_query("“weight of evidence”") == '"weight of evidence"'
     assert utils.remove_unquoted_stop_words("the") == "the"
-    assert utils.remove_unquoted_stop_words("'the'") == "'the'"
+    assert utils.remove_unquoted_stop_words('"the"') == '"the"'
```

Comment:
This line looks like Black has chewed it up a bit, we might need an ignore statement.

Reply:
aaaah yep well spotted - i'll add.